picard/browser/browser.py

```diff
@@ -142,9 +146,26 @@ def stop(self):
         else:
             log.debug("Browser integration inactive, no need to stop")
 
+        if stop_url_handler:
+            QDesktopServices.unsetUrlHandler(PICARD_CUSTOM_PROTOCOL)
+
+    def url_handler(self, url: QtCore.QUrl):
+        """URL handler used for custom protocol handling."""
+        log.debug("Received custom URL: %s", url.toString())
+
+        if url.scheme() != PICARD_CUSTOM_PROTOCOL:
```

Comment:
I wonder if  `PICARD_PROTOCOL_SCHEME` isn't a better name since we match a scheme (and it's the scheme part of the URI).

Reply:
Yes, better. I'll change this